pkg/deploy/gateway/traefik_config.go

```diff
@@ -52,7 +52,9 @@ type TraefikConfigStripPrefix struct {
 }
 
 type TraefikConfigForwardAuth struct {
-	Address string `json:"address"`
+	Address            string            `json:"address"`
+	TrustForwardHeader bool              `json:"trustForwardHeader"`
```

Comment:
We're using ForwardAuth in workspace routing configuration. Since this is pure `bool`, I guess it will become `false` value when not specified. Is it ok?

Reply:
I'm reading https://doc.traefik.io/traefik/middlewares/http/forwardauth/#trustforwardheader such that this defaults to `false`, so yes, this should be ok.